### DISCUSSION TITLE
Blob Storage Connection

### DISCUSSION THREAD

**Discussion by pri-kise** [microsoft/ALAppExtensions#16653] [Ideas]
I really like the new System Application feature to easy upload files into azure blob storage.
Right now we are creating for different independent apps the same setup structure to store Credentials to connect with the Azure Blob API.

Are there any plans to provide a Setup Table to create and save different Connections that later can be used by any App?

I also thought about adding a simple setup table for storing different credentials for the Azure Blob API, but I wasn't sure if this is something Microsoft has planned for the Application....?

**Comment by JesperSchulz** [comment]
We don't have any concrete plans quite yet. But we are looking into, which plans we should make for file handling in the application. But seeing that we haven't even started wrapping our minds around that, it will be some time before anything materializes in the product.
